x-pack/plugins/observability/server/lib/rules/slo_burn_rate/executor.ts

```diff
@@ -94,11 +94,11 @@ export const getRuleExecutor = ({
     const { dateEnd } = getTimeRange('1m');
     const results = await evaluate(esClient.asCurrentUser, slo, params, new Date(dateEnd));
 
-    if (results.length > 0) {
-      const alertLimit = alertFactory.alertLimit.getValue();
-      let hasReachedLimit = false;
-      let scheduledActionsCount = 0;
+    const alertLimit = alertFactory.alertLimit.getValue();
```

Comment:
@shahzad31 FYI I see you recently fixed https://github.com/elastic/kibana/pull/173379. I had to move a few things around.